clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -722,7 +724,83 @@ describe("Fides-js TCF", () => {
           ).to.eql(true);
         });
       });
-
+      it("skips saving preferences to API when custom API fn is provided in Fides.init", () => {
+        cy.fixture("consent/experience_tcf.json").then((privacyExperience) => {
+          stubConfig({
+            options: {
+              isOverlayEnabled: true,
+              tcfEnabled: true,
+              api: {
+                /* eslint-disable @typescript-eslint/no-unused-vars */
+                savePreferencesFn: (
+                  fides: FidesConfig,
+                  consent: CookieKeyConsent,
+                  fides_string: string | undefined,
+                  experience: PrivacyExperience
+                ) => {},
```

Comment:
You should add an explicit assertion to ensure this save function gets called. That can be done with some Cypress spying